include/signalrclient/signalr_value.h

```diff
@@ -180,8 +180,20 @@ namespace signalr
             double number;
             std::map<std::string, value> map;
 
+            // constructor is not implicitly called
+#pragma warning (push)
+#pragma warning (disable: 4582)
```

Comment:
Has this false-positive been fixed? https://developercommunity.visualstudio.com/content/problem/564332/stdoptional-generates-warning-c4582-constructor-is.html

Also, do these pragmas only apply to msvc?

Reply:
It's not a false positive.

> Also, do these pragmas only apply to msvc?

Yes